osquery/tables/system/windows/system_util.h

```diff
@@ -24,19 +24,21 @@
 
 namespace osquery {
 namespace tables {
- 
+
 class WmiResultItem {
  public:
-  explicit WmiResultItem(IWbemClassObject *result) : result_(result) {};
+  explicit WmiResultItem(IWbemClassObject* result) : result_(result){};
```

Comment:
So many functions without documentation! Not your problem or the goal of this PR perhaps.